### PR TITLE
added some tests

### PR DESCRIPTION
**Pull request by 5saviahv**
* eslint ignore jest/expect-expect
* I tried add some diversity with tests

### CODE REVIEW

File: test/cheerio.js [hunk 1]
```diff
@@ -369,4 +376,33 @@ describe('cheerio', function () {
...
+      expect(utils.cssCase('neverSayNever')).toBe('never-say-never');
+      expect(utils.cssCase('CSSCase')).toBe('-c-s-s-case');
+    });
+
+    it('cloneDom : should be able clone single Elements', function () {
+      var result = [];
+      utils.domEach(cheerio('<p>A</p><p>B</p>'), function (i, el) {
+        result = result.concat(utils.cloneDom(el));
+      });
+      expect(result).toHaveLength(2);
```
Comment (fb55) [inline test/cheerio.js:399]: Can we check the actual shape of the response here?

File: test/api/attributes.js [hunk 2]
```diff
@@ -359,7 +370,7 @@ describe('$(...)', function () {
         balls: 'giandor',
       });
       // Adding as string.
-      var b = $('.linth').data('snack', 'chocoletti');
+      var b = $('.linth').data('snack', 'chocoletti').data(4, 'chocoletti');
```
Comment (fb55) [inline test/api/attributes.js:373]: Check the value of `data(4)`?

**Review by fb55 — COMMENTED** [review]
Nice additions! Two comments, otherwise this is great.

File: test/api/attributes.js (hunk 2 above)
Comment (5saviahv) [reply test/api/attributes.js:373]: yes I used not a string for test. and it doesnt break.

It managed to throw when I did same with setAttr.

Comment (5saviahv) [reply test/api/attributes.js]: I added throw check

File: test/cheerio.js (hunk 1 above)
Comment (5saviahv) [reply test/cheerio.js]: I wanted test it with toEqual but because circular reference it will hang jest, so I removed it. But I added some tests.

**Comment by fb55** [comment]
Great PR 👏

**Pull request merged**